### DISCUSSION TITLE
蓝灯祝贺中国的朋友们中秋节快乐! Happy Mid-Autumn Festival

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#872] [General]
尊敬的Lantern VPN用户们：

在这个团圆美满的中秋佳节，Lantern VPN团队衷心祝愿您和您的家人月圆人圆，幸福安康！无论您身处何地，愿我们为您提供的服务如明亮的灯塔，照亮您的每一个网络旅程。愿您度过一个祥和愉快的中秋节，阖家团聚，万事如意！

中秋快乐，阖家幸福！

Lantern VPN团队

![黄紫色线描漫画嫦娥玉兔可爱中秋节节日宣传中文海报](https://github.com/user-attachments/assets/ca1ffd10-10ca-44bf-89a5-ba7928c29cae)

**Comment by qian6361** [comment]
你好我也是买了专业版，但用不了哇。麻烦看看呢，邮箱zhangqian6361@gmail.com

**Comment by MaxMKMax** [comment]
@qian6361 您好. 请您也提供我支付单据. 这样我才能帮您提交问题上去. 谢谢您!

**Comment by qian6361** [comment]
Mike K ***@***.***> 于2024年9月26日周四 02:40写道：

**Comment by qian6361** [comment]
[image: 支付单据.jpg]

zhang qian ***@***.***> 于2024年9月30日周一 10:31写道：

**Reply by MaxMKMax** [reply]
@qian6361 您好, 您的支付单据被Github屏蔽了. 您发到我的邮箱给我吧. 我帮您提交问题. 谢谢!

**Comment by qian6361** [comment]
提供了这么久，也没反应啊。能不能处理嘛

Mike K ***@***.***> 于2024年10月1日周二 04:10写道：

**Reply by MaxMKMax** [reply]
@qian6361 您好. 请问您是用哪个邮箱发给我的? 我邮箱里查不到您的支付单据. 您的支付单据被Github屏蔽了. 您发到我的邮箱给我吧. 
mike@getlantern.org

**Comment by qian6361** [comment]
就这个邮箱哇 用户名也是这个邮箱，重新附了一份，您看看现在能不能看到呢

Mike K ***@***.***> 于2024年10月12日周六 00:50写道：

**Reply by MaxMKMax** [reply]
@qian6361 
您好. 这是您的激活码:
MF7DP-MD7V8-V8J26-Q2R2F-K2QRV

**Comment by qian6361** [comment]
[image: image.png]显示 兑换激活码时出错

Mike K ***@***.***> 于2024年10月24日周四 00:36写道：
